test/build/package-tippy-js/expected-build/web_modules/import-map.json

```diff
@@ -0,0 +1,6 @@
+{
+  "imports": {
+    "tippy.js": "./tippy.js.js",
```

Comment:
blarg, `.js.js` is something I've been really trying to avoid.

How about instead:

```
foo -> web_modules/foo.js
tippy.js -> web_modules/tippyjs.js
tippy.js/style.css -> web_modules/tippyjs/style.css
```

Reply:
I know it looks weird, but this seems correct to me? Because the first `.js` is just an arbitrary part of the package name beyond our control.

Reply:
I think there's technically correct and then there's human correct. `tippy.js.js` is technically correct but looks like a bug to me. 

If it helps, we were already renaming npm packages! Rollup currently installs both `tippy` and `tippy.js` to `web_modules/tippy.js`. One of the reasons I like the suggestion above is that while `tippy + tippy.js` in the same project is rare, `tippyjs + tippy.js` in the same project would be downright ludicrous.

Reply:
OK just pushed changes to that generates the following:

```
├── tippyjs/
│   ├── dist/tippy.css
│   └── dist/tippy.css.proxy.js
└── tippy.js
```

This weird rewriting should only happen on 2 conditions:
1. this is a web module (e.g. loaded from npm)
1. the web module only ends in `.js` (we should only realistically have that file extension at the top-level in `web_modules`; everything else like `.css` will be in sub-folders)

Reply:
how do you feel about `tippyjs.js` being the name of the JS file, as well? I do agree with your original sentiment about being explicit and easy to reason about, and I think I'd find `tippyjs.js` & `tippyjs/*` easier to reason about as a user.

Reply:
🍌. Updated!